packages/esm-appointments-app/src/config-schema.ts

```diff
@@ -47,6 +48,16 @@ export const configSchema = {
     _description: 'Display for the privilege to view and manage appointments',
     _default: "Today's Appointments Widget: Display Full View",
   },
+  bahmniAppointmentsUiBaseUrl: {
+    _type: Type.String,
+    _description: 'Configurable BahmniUi base URL',
+    _default: '/appointments',
+  },
+  appointmentsBaseUrl: {
+    _type: Type.String,
+    _description: 'Configurable an alternative URL for the appointments widget',
```

Comment:
Configurable alternative URL for the Appointments app. Eg, the Bahmni Appointments UI URL